### PR TITLE
[doc][improve] update concepts-messaging.md

### PR DESCRIPTION
**Pull request by DaveDuggins**
### Motivation

Updated concepts-messaging.md with new screen shots. Minor edits to ensure text matches.

### Modifications

Edited concepts-messaging.md

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

### Documentation

- [ ] `doc-required` 
(Your PR needs to update docs and you will update later)

- [ ] `doc-not-needed` 
(Please explain why)

- [x] `doc` 
(Your PR contains doc changes)

- [ ] `doc-complete`
(Docs have been already added)

### CODE REVIEW

File: site2/docs/concepts-messaging.md [hunk 1]
```diff
@@ -170,6 +184,8 @@ If the consumer fails to receive all chunks of a message within a specified peri
 
 A consumer is a process that attaches to a topic via a subscription and then receives messages.
 
+![Consumer](/assets/consumer.svg
```
Comment (Anonymitaet) [inline site2/docs/concepts-messaging.md:187]:
```suggestion
![Consumer](/assets/consumer.svg)
```

**Review by Anonymitaet — COMMENTED** [review]
PTAL

**Comment by DaveDuggins** [comment]
Attaching .svg files.
pub-sub-border
delayed-delivery
message-deduplication
retention-expiry
consumer
batching

File: site2/docs/concepts-messaging.md [hunk 2]
```diff
@@ -958,3 +974,7 @@ The following is an example of delayed message delivery for a producer in Java:
 producer.newMessage().deliverAfter(3L, TimeUnit.Minute).value("Hello Pulsar!").send();
 ```
 
+![consumer](https://user-images.githubusercontent.com/94193423/192619010-cb1a5772-f0b8-4d40-b9f9-1df1c5d8ce68.svg)
```
Comment (momo-jun) [inline site2/docs/concepts-messaging.md:977]: @DaveDuggins I guess your intention is to commit the image files through this pull request, so the last 4 lines are not required in the content update. You only need to add the SVG files to your branch.

Comment (DaveDuggins) [reply site2/docs/concepts-messaging.md:1009]: Understood.  Thanks Jun!

**Comment by DaveDuggins** [comment]
Local preview looks good. Note that all preview images are for the same page.

Delayed delivery
Persistent messaging
Namespace
Key shared
Shared
Failover
Exclusive
Subscription
Consumer
Batching

**Review by merlimat — APPROVED** [review]
👍

File: site2/docs/concepts-messaging.md [hunk 3]
```diff
@@ -87,17 +89,29 @@ You can set producer access mode through Java Client API. For more information,
...
+    .topic(“topic-name”) 
+    .compressionType(CompressionType.LZ4) 
+    .create();
+```
 
 ### Batching
 
 When batching is enabled, the producer accumulates and sends a batch of messages in a single request. The batch size is defined by the maximum number of messages and the maximum publish latency. Therefore, the backlog size represents the total number of batches instead of the total number of messages.
 
+![Batching](/assets/batching.svg)
```
[the remark this answers is not in the document]
Comment (tisonkun) [reply site2/docs/concepts-messaging.md:113]: Create issue at https://github.com/apache/pulsar/issues/17960. New patch can link there.

**Pull request merged**